apps/docs/content/components/Grid/index.mdx

```diff
@@ -0,0 +1,286 @@
+    <img
+      src="https://github.com/primer/brand/assets/912236/0703ba43-7f47-4ad1-bc1d-020bc095e362"
+      alt=""
+    />
+    <Caption>
+      Don't use different column spans when grouping the same element type.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+In some cases, you may need specific amount of columns or rows instead of an even distribution. Use different column spans when creating a visual hierarchy between items or when defining a page or section layout structures. E.g. A section with an [image](/components/Image) and [text](/components/text), a form with inputs and text areas, or a layout with main content and a sidebar.
+
+![](https://github.com/primer/brand/assets/912236/bc442c41-3546-4341-bab3-1c649ccfe7d2)
+
+<DoDontContainer stacked>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/912236/83e4eedf-5cc2-4dcc-a9dd-48d563744dd2"
+      alt=""
+    />
+    <Caption>
+      Use different column spans when you want to create a visual hierarchy
+      between different elements.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/912236/911a6b4b-791a-4186-8079-6296e17c65d0"
+      alt=""
+    />
+    <Caption>
+      Don't use equal column spans when the content is not related or the
+      elements are different.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Responsive
+
+The grid component provides control over the number of columns at each breakpoint. For most of the cases, we recommend using the same column span on each child at each breakpoint to create a consistent layout and spacing in all screen sizes.
+
+For example, a group of cards might span 3 columns on a extra large viewport, 4 columns on large viewports, 6 on medium and 12 columns on small viewports.
+
+<div>
+  <CustomVideoPlayer
+    width="100%"
+    src="https://github.com/primer/brand/assets/912236/a99beba9-b22f-4e2b-a5fa-66c2db246965"
+  />
+  <Caption>
+    Use the same column span for each content unit at each breakpoint to create
+    visually consistent layouts.
+  </Caption>
+</div>
+
+When using even column spans, we recommend to span each item full width on small viewports and stacking the items vertically to ensure that the content is readable and accessible on small screens.
+
+<div>
+  <CustomVideoPlayer
+    width="100%"
+    src="https://github.com/primer/brand/assets/912236/cd0d4335-03b2-48f2-98f0-2c53ef61544b"
+  />
+  <Caption>
+    Use full width on small viewports to ensure that the content is readable and
+    accessible.
+  </Caption>
+</div>
+
+## Grid vs Stack
+
+We recommend using the grid or the [stack](/components/Stack) component depending on how the child items are displayed and how are they expected to work on various screen sizes. If the element has all of its child items displayed inline, then most probably the [stack](/components/Stack) component would be the best option. However, if you think of columns and rows, then the use the grid component[^2].
```

Comment:
```suggestion
We recommend using either the grid or [stack](/components/Stack) component, depending on how the child items are displayed and how they should work on different screen sizes. If the element has all of its child items displayed inline, the stack component is probably the best option. Otherwise, if you're thinking of columns and rows, use the grid component.
[^2].
```

Reply:
+1 to ☝️ 

Also adding that we should use more active voice here, and remove ambiguity by saying probably if we can. 

The use-case for Stack is around sequential lists that appear across _either_ the x and y axis.. whereas Grids present can present across both. Grid is also preferred for general page layout, unlike Stack which doesn't have a built-in max-width.